contracts/payloads/IGP43/PayloadIGP43.sol

```diff
@@ -0,0 +1,636 @@
+        uint8 supplyDecimals;
+        uint8 borrowDecimals;
+        uint256 vaultId;
+        bytes32 liquiditySupplyExchangePriceSlot;
+        bytes32 liquidityBorrowExchangePriceSlot;
+        bytes32 liquidityUserSupplySlot;
+        bytes32 liquidityUserBorrowSlot;
+    }
+
+    function constantsView() external view returns (ConstantViews memory constantsView_);
+}
+
+contract PayloadIGP43 {
+    struct VaultConfig {
+        uint256 vaultId;
+        address supplyToken;
+        address borrowToken;
+        uint8 supplyMode;
+        uint256 supplyExpandPercent;
+        uint256 supplyExpandDuration;
+        uint256 supplyBaseLimitInUSD;
+        uint256 supplyBaseLimit;
+        uint8 borrowMode;
+        uint256 borrowExpandPercent;
+        uint256 borrowExpandDuration;
+        uint256 borrowBaseLimitInUSD;
+        uint256 borrowBaseLimit;
+        uint256 borrowMaxLimitInUSD;
+        uint256 borrowMaxLimit;
+    }
+
+    struct TokenConfig {
+        uint256 baseDebtCeiling;
+        uint256 maxDebtCeiling;
+        uint256 baseWithdrawalLimit;
+    }
+
+    struct DexConfig {
+        uint256 dexId;
+        TokenConfig token0Config;
+        TokenConfig token1Config;
+    }
+
+    uint256 public constant PROPOSAL_ID = 42;
+
+    address public constant PROPOSER = 0xA45f7bD6A5Ff45D31aaCE6bCD3d426D9328cea01;
+
+    IGovernorBravo public constant GOVERNOR = IGovernorBravo(0x0204Cd037B2ec03605CFdFe482D8e257C765fA1B);
+    ITimelock public constant TIMELOCK = ITimelock(0x2386DC45AdDed673317eF068992F19421B481F4c);
+
+    address public constant TEAM_MULTISIG = 0x4F6F977aCDD1177DCD81aB83074855EcB9C2D49e;
+
+    address public immutable ADDRESS_THIS;
+
+    IFluidLiquidityAdmin public constant LIQUIDITY = IFluidLiquidityAdmin(0x52Aa899454998Be5b000Ad077a46Bbe360F4e497);
+
+    address public constant ETH_ADDRESS = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+
+    FluidDexReservesResolver public constant DEX_RESERVES_RESOLVER = FluidDexReservesResolver(0x278166A9B88f166EB170d55801bE1b1d1E576330);
+
+    FluidVaultFactory public constant VAULT_FACTORY = FluidVaultFactory(0x324c5Dc1fC42c7a4D43d92df1eBA58a54d13Bf2d);
+
+    address internal constant wstETH_ADDRESS = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+    address internal constant USDT_ADDRESS = 0xdAC17F958D2ee523a2206206994597C13D831ec7;
+    address internal constant USDC_ADDRESS = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address internal constant weETH_ADDRESS = 0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee;
+    address internal constant cbBTC_ADDRESS = 0xcbB7C0000aB88B473b1f5aFd9ef808440eed33Bf;
+
+
+    address public constant wstETH_ETH_Dex = 0x6d83f60eEac0e50A1250760151E81Db2a278e03a;
+    address public constant USDC_USDT_Dex = 0x5014E47Ca35eC6963985736846a4bCcd68Fab2F2;
+    address public constant cbBTC_WBTC_Dex = 0x77545D43dFb6F12f5c19034E2a3167e18265F392;
+
+    address public constant wstETH_ETH_SMART_COL_DEBT = 0x57fed7c9b3c763999c519264931790cBcA331417;
```

Comment:
```suggestion
    address public constant wstETH_ETH_SMART_COL_DEBT = getVaultAddress(34);
```

Add getVaultAddress helper function: https://github.com/Instadapp/inst-governance/blob/main/contracts/payloads/IGP41/PayloadIGP41.sol#L843

This will simplify the verification process of addresses. 

Do the same for above dexes